setup.cfg

```diff
@@ -22,13 +23,9 @@ exclude = .git,
     versioneer*,
     examples/*,
     # TODO: remove the items below and fix linting issues
-    great_expectations/checkpoint,
-    great_expectations/jupyter_ux,
-    great_expectations/cli,
-    great_expectations/expectations,
-    great_expectations/dataset,
-    great_expectations/render,
-    scripts/*,
+    great_expectations/cli, # 11 warnings
+    great_expectations/expectations, # 122 warnings
+    great_expectations/dataset, # 52 warnings
```

Comment:
Only 3 modules left.

Will have a followup PR for CLI shortly. Doing that one separately because I'll be doing type coverage as well.